packages/web-app-files/src/components/SideBar/Actions/SpaceActions.vue

```diff
@@ -50,6 +49,7 @@ import {
   useSpaceActionsRestore
 } from 'web-pkg/src/composables/actions/spaces'
 import { useSpaceActionsUploadImage } from '../../../composables/actions/spaces/useSpaceActionsUploadImage'
+import { useFileActionsDownloadArchive } from 'web-app-files/src/composables'
```

Comment:
```suggestion
import {
  useFileActionsDownloadArchive,
  useSpaceActionsUploadImage
} from 'web-app-files/src/composables'
```

... and delete the line above